reactor-netty-core/src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -817,5 +817,7 @@ interface MeterRegistrar {
 
 		void registerMetrics(String poolName, String id, SocketAddress remoteAddress, ConnectionPoolMetrics metrics);
 
+		void deRegisterMetrics(String poolName, String id, SocketAddress remoteAddress);
```

Comment:
Let's make JAPICMP happy (I will remove the default implementation in Reactor Netty version 2.0.x where we can make this incompatible change)

```
Japicmp report was filtered and interpreted to find the following incompatibilities:
source incompatible change: **** MODIFIED INTERFACE: PUBLIC ABSTRACT STATIC reactor.netty.resources.ConnectionProvider$MeterRegistrar  (not serializable)
source incompatible change:     +++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void deRegisterMetrics(java.lang.String, java.lang.String, java.net.SocketAddress)
```

```suggestion
		/**
		 * Invoked when a connection pool is disposed.
		 *
		 * @param poolName the pool name
		 * @param id the pool id
		 * @param remoteAddress the remote address
		 * @since 1.0.26
		 */
		default void deRegisterMetrics(String poolName, String id, SocketAddress remoteAddress) {
		}
```

Reply:
Sounds good, thanks!

Reply:
I removed the default implementation in Reactor Netty 2 with PR #2613